app/javascript/packages/document-capture/components/in-person-full-address-search.tsx

```diff
@@ -186,14 +191,21 @@ function FullAddressSearch({
         />
       </ValidatedField>
       <ValidatedField ref={validatedStateFieldRef}>
-        <TextInput
+        <SelectInput
           required
           ref={registerField('state')}
           value={stateInput}
           onChange={onStateChange}
           label={t('in_person_proofing.body.location.po_search.state_label')}
           disabled={disabled}
-        />
+        >
+          <option value="" disabled selected>
```

Comment:
It's not clear to me how React would handle both passing the `value` for the `<select>` as well as assigning `selected` to the default option. Is it not enough to pass the value as `''` in the initial state?

Reply:
fixed in 128d2f3343e8e3ac2890b0a28f07238c97ff1c2b